### PR TITLE
Add the page for displaying the configuration list

### LINKED ISSUE #1528: Support configuration management on the backend.
### What would you like to be added?

Add configuration management backend services

### Why is this needed?

Enables users to configure plug-ins on the backend.

### PR DESCRIPTION
**Pull request by hanbingleixue**
**What type of PR is this?**

Feature.

**What this PR does / why we need it?**

Add the page for displaying the configuration list.
1. The configuration list page is added to display the configuration list information by page.
2. Plug-in configurations and other configurations can be queried on the page. The query can be performed based on conditions such as the application name and service name.
3. Supports configuration deletion on the page.
4. Users can switch to other pages to add or modify configurations.
5. The page rendering is as follows

image

**Which issue(s) this PR fixes？**

Fixes #1528 

**Does this PR introduce a user-facing change?**

No

## Checklist
- [x] Make sure there is a [GitHub_issue](https://github.com/sermant-io/Sermant/issues) related with this PR before you start working on it.
- [x] Make sure you have squashed your change to one single commit.
- [x] GitHub Actions works fine in this PR.

### CODE REVIEW

**Comment by lilai23** [comment]
Beautify the configuration center display and configure the search box

**Comment by lilai23** [comment]
Change the edit button to a view details button

**Comment by lilai23** [comment]
Add front-end filtering

File: sermant-backend/src/main/webapp/frontend/src/views/ConfigView.vue [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+  appName: string,
+  serviceName: string,
+  environment: string,
+  zone: string,
+  group: string,
+  key: string,
+}
+
+const fetchData = () => {
+  // 根据当前页码和每页显示条数，截取当前分页的数据
```
Comment (daizhenyu) [inline sermant-backend/src/main/webapp/frontend/src/views/ConfigView.vue]: Comments in English

Comment (hanbingleixue) [reply sermant-backend/src/main/webapp/frontend/src/views/ConfigView.vue]: done

**Review by daizhenyu — DISMISSED** [review]

**Review by lilai23 — APPROVED** [review]

**Review by daizhenyu — APPROVED** [review]

**Pull request merged**